src/popup/utils/utils.ts

```diff
@@ -336,6 +336,10 @@ export const calculateNameClaimFee = (name: string): BigNumber => calculateFee(
   },
 );
 
+export function urlWithParams(url: string, params: Record<string, any>) {
```

Comment:
Functions executes code so their names should have imperative form. Like `do...`, `create...`, `fetch...`. I suggest to rename this function to `addParamsToUrl`.